api_fhir_r4/converters/claimConverter.py

```diff
@@ -26,13 +28,22 @@ def to_fhir_obj(cls, imis_claim):
         fhir_claim = FHIRClaim()
         cls.build_fhir_pk(fhir_claim, imis_claim.uuid)
         fhir_claim.created = imis_claim.date_claimed.isoformat()
-        fhir_claim.facility = HealthcareServiceConverter.build_fhir_resource_reference(imis_claim.health_facility)
+        if imis_claim.health_facility is None:
+            raise FHIRRequestProcessException([F'Cannot construct a {imis_claim.uuid} claim if HF is None'])
```

Comment:
Exception message strange: 'Cannot construct a UUID claim if HF is None'. Need to be updated.

Reply:
Removed raising exception as the absence of HF is not possible (SQL foreign key constraint)